services/web-app/pages/contributing/schema.mdx

```diff
@@ -398,7 +429,7 @@ designKits:
 | `license`     | Design kit license.                                                                                                                                                                                                                                                           | Optional | String           | –       | `apache-2.0`, `ibm-internal`, `mit`                                      |
 | `noIndex`     | If set to `true`, the global catalogs will exclude the kit.                                                                                                                                                                                                                   | Optional | Boolean          | `false` | –                                                                        |
 
-#### Design kit type
+### Design kit type
 
 The design kit `type` key can have the following values:
```

Comment:
For the intro text to design kits, Morgana was suggesting we mention specific tools. i.e. Design Kits are libraries from Figma and Sketch—and other tools such as Adobe XD and Invision. They are the design counterpart to coded assets and contain components and elements as well as wireframes and guidance to aid designers. Design kits are indexed in the [carbon-platform](https://github.com/carbon-design-system/carbon-platform) repository 
[carbon.yml](https://github.com/carbon-design-system/carbon-platform/blob/main/packages/resources/carbon.yml)
 data file.